### PR TITLE
fix(authoring): Removed tags from item body before sending it to alch…

### PR DESCRIPTION
**Pull request by eozan**
…emy.

SDPA-156

### CODE REVIEW

**Comment by mugurrus** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/metadata/metadata.js [hunk 1]
```diff
@@ -277,7 +277,12 @@ function MetadataTagsDirective(api) {
              */
             scope.refresh = function() {
                 scope.refreshing = true;
-                api.save('keywords', {text: scope.item[scope.sourceField]})
+                var body = scope.item[scope.sourceField]
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: pls add some unittests

File: client/app/scripts/superdesk-authoring/metadata/metadata.js [hunk 2]
```diff
@@ -277,7 +277,12 @@ function MetadataTagsDirective(api) {
...
+                    .replace(/<br[^>]*>/gi, '&nbsp;')
+                    .replace(/<\/?[^>]+><\/?[^>]+>/gi, ' ')
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: why is this line needed?

**Comment by pavlovicnemanja** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/metadata/metadata.js (hunk 1 above)
Comment (eozan) [inline client/app/scripts/superdesk-authoring/metadata/metadata.js]: done
